monkey/monkey_island/cc/ui/src/components/report-components/RansomwareReport.js

```diff
@@ -14,17 +14,62 @@ class RansomwareReport extends React.Component {
           <p>
             This report shows information about the ransomware simulation run by Infection Monkey.
           </p>
+          {this.getExploitationStats()}
         </div>
     )
   }
 
+  getExploitationStats() {
+    let num_scanned = this.props.report.propagation_stats.num_scanned_nodes;
+    let num_exploited = this.props.report.propagation_stats.num_exploited_nodes;
+
+    return (
+      <div>
+        <p>
+          The Monkey discovered <span className='badge badge-warning'>{num_scanned}</span> machines
+          and successfully breached <span className='badge badge-danger'>{num_exploited}</span> of them.
+        </p>
+        {this.getExploitationStatsPerExploit()}
+      </div>
+    )
+  }
```

Comment:
![image](https://user-images.githubusercontent.com/19957806/125282340-43ee2200-e2e5-11eb-8968-da26d829a837.png)


The report description and the report details kind of run together. We should put some kind of header here to separate them. I think "Propagation" is sufficient for now.

Reply:
![image](https://user-images.githubusercontent.com/19957806/125325850-6ac14e00-e30f-11eb-9fc4-05aa6da918a6.png)